### DISCUSSION TITLE
Consider creating a Contribution Token

### DISCUSSION THREAD

**Discussion by joelamouche** [kodadot/nft-gallery#3410] [General]
So there are many ways to go about this and the flexibility is what makes it so interesting...
For example you could create an ERC20 on Moonriver and start rewarding people who contribute to Kodadot.
What is interesting is that you can reward tiny tasks for which you wouldn't necessarily pay people: opening issues, giving UX feedback, talking in a community call, creating an NFT, trying a new feature, etc.
As for the incentive, you can chose. Of course as a contributor I would like to see this token's value correlated to the companie's success but that is a decision you will have to make. An easier incentive is to give out Tshirts, Dotsama Prague tickets and NFTs to token holders. you could also let them vote on some UX decisions (because this token will also be akin to a reputation token)

**Comment by yangwao** [comment]
Amazing someone is popping up with this. I've bit dropped my thoughts back in 12/2021, section [Koda-Token](https://github.com/kodadot/nft-gallery/discussions/1424) 
- https://docs.kodadot.xyz/token/draft.html#tokenomics-kodadot-token-kodao
 
Both are just drafts, so it needs a bit of rework. 
I guess the hardest is to fine-tune proportions per contributions, review etc 

> For example you could create an ERC20 on Moonriver and start rewarding people who contribute to Kodadot.
Yes, this or at Statemine was the original idea to be more Substrate native. 
 
The whole point of having project tokens is to expose contributors to potential non-linear rewards based on their contributions to Kodadot and future success of the course as seems we've got fair traction in development there. Of course, it might attract more new contributors on the scale as we've already laid out there a quite steady process

I would be more than happy to see other's ideation on this topic so we can push $kodao token forward :)
